### PR TITLE
Fix for the selection not working on tablet

### PR DESCRIPTION
**Pull request by sekwah41**
See issue #35

### CODE REVIEW

File: src/selection.js [hunk 1]
```diff
@@ -333,6 +333,8 @@
         },
 
         cancel: function() {
+            this.outerTracker.setTracking(false);
```
Comment (iangilman) [inline src/selection.js]: Might be good to add a comment for why this is here, since it's not obvious just by looking at the code.

Comment (sekwah41) [reply src/selection.js]: Fair point, will do tomorrow as I am busy today

**Comment by sekwah41** [comment]
That should be the fix with comments for now. Will look further into the issue as soon as I get some free time to do so

**Comment by iangilman** [comment]
Looks great to me! I don't have permissions to merge pull requests on this repository... @picturae, can you merge it (if it looks good to you)? @MindFreeze do you have permission to merge?

**Pull request merged**